### PR TITLE
[Habana] add a unit test to stress gather

### PR DESCRIPTION
**Pull request by hyuen**
Summary: add a unit test to stress gather

Test Plan: test is disabled until Gather is fixed

### CODE REVIEW

File: tests/unittests/HabanaBackendTest.cpp [hunk 1]
```diff
@@ -1711,3 +1711,52 @@ TEST_F(HabanaBackendTest, DISABLED_BatchedGather) {
...
+    // fill up the ID list with random data
+    for (unsigned int i = 0; i < input_ids.size(); i++) {
+      input_ids[i] = rand() % M;
+    }
+    indices_h = input_ids;
+
+    EE_.run(ctx_);
+
+    auto H = ctx_.get(result->getPlaceholder())->getHandle();
+    for (unsigned i =0; i < id_len; i++) {
```
Comment (rdzhabarov) [inline tests/unittests/HabanaBackendTest.cpp]: does clang format like this? (i =0)

File: tests/unittests/HabanaBackendTest.cpp [hunk 2]
```diff
@@ -1711,3 +1711,52 @@ TEST_F(HabanaBackendTest, DISABLED_BatchedGather) {
   EXPECT_FLOAT_EQ(H.at({2, 0}), 4.5);
   EXPECT_FLOAT_EQ(H.at({2, 1}), 1.2);
 }
+
+TEST_F(HabanaBackendTest, DISABLED_BatchedGatherEx) {
+  unsigned M = 1000;
+  unsigned N = 1;
+
+  // Fill out the array with random data
+  std::vector<float> input_data;
```
Comment (rdzhabarov) [inline tests/unittests/HabanaBackendTest.cpp]: could you fix the naming here. The convention is to use: inputData, InputIds etc.

File: tests/unittests/HabanaBackendTest.cpp (hunk 1 above)
Comment (bertmaher) [reply tests/unittests/HabanaBackendTest.cpp]: Tip: `utils/format.sh fix` will clang-format the whole tree.

**Comment by rdzhabarov** [comment]
@hyuen I added that test as part of enabling gather - https://github.com/pytorch/glow/pull/2867

**Comment by rdzhabarov** [comment]
closing since that was merged as part of #2867

**Pull request closed without merging**